packages/grammy-nestjs/src/types/grammy-types.ts

```diff
@@ -13,7 +13,7 @@ export type StringOrRegexQuery = string | RegExp | string[]
 
 export type CommandQuery = 'start' | 'help' | 'settings' | 'admin'
 
-export type EmitterQuery = 'on' | 'command' | 'hears' | 'gameQuery' | 'inlineQuery' | 'callbackQuery'
+export type BotQuery = 'on' | 'command' | 'hears' | 'gameQuery' | 'inlineQuery' | 'callbackQuery'
```

Comment:
Do these make sense, given that `string` is also permitted in the union type?

Perhaps https://github.com/grammyjs/grammY/blob/bfed670cae3d06035d75cebae5035c92902c1f74/src/composer.ts#L7 will be useful.

Reply:
I don't think so -- these are the only ones I'm supporting right now.  Does that make sense?